pkg/util/fsnotification/fsnotification.go

```diff
@@ -0,0 +1,52 @@
+package fsnotification
```

Comment:
Missing package documentation :)

Names, names, ... importing both `fsnotify` and `fsnotification` looks really funny.